push/urls.py

```diff
@@ -6,13 +6,13 @@
     url(r'^$', push_views.Landing.as_view(), name="push.landing"),
 
     url(r'^apps/$',
-        push_views.PushApplications.as_view(),
+        push_views.List.as_view(),
         name='push.applications'),
 
     url(r'^apps/(?P<pk>[0-9]+)/$', push_views.Details.as_view(),
         name='push.details'),
 
     url(r'^apps/(?P<pk>[0-9]+)/validate/$',
```

Comment:
Could we change this to `apps/{pk}/validation` too? So we don't have a verb in the resource identifier? We should also update the `validation.html` template to show something like "You already validated this app key." Instead of showing them a form to validate the app key again.